### DISCUSSION TITLE
Add text to a PDF

### DISCUSSION THREAD

**Discussion by premins** [pikepdf/pikepdf#472] [Q&A]
Hi! I'm trying to add text to an existing PDF. I thought a watermark would be the solution but the documentation on watermarks (https://pikepdf.readthedocs.io/en/latest/topics/overlays.html) seem to talk about adding a pdf within another pdf. Is it possible to only add a string as the watermark? 
Thanks!

**Comment by jbarlow83** [comment]
To add text it would be easiest to render the text using reportlab and then use pikepdf to overlay it.
